jwst/photom/photom.py

```diff
@@ -903,9 +903,13 @@ def photom_io(self, tabdata, order=None):
                 if unit_is_surface_brightness:
                     slit.meta.bunit_data = 'MJy/sr'
                     slit.meta.bunit_err = 'MJy/sr'
+                    self.input.meta.bunit_data = None
```

Comment:
would be good to include a code comment here to indicate that we're intentionally blanking out the values in the top level of the meta tree, so that units info only comes from the individual slit instances.

Reply:
comment added